src/clusterfuzz/_internal/bot/fuzzers/utils.py

```diff
@@ -30,7 +30,7 @@
 EXTRA_BUILD_DIR = '__extra_build'
 
 
-def is_fuzz_target_local(file_path, file_handle=None):
+def is_fuzz_target_local(file_path, file_opener=None):
```

Comment:
The fact that this could return true for remote zip archive is confusing me. Can you explain a bit more about what this code is doing and how it's used?

Reply:
Yeah, I think the function name is not really correct, since it can operate on a `file_handle`, which might not necessarily be local.